src/main/java/Application.java

```diff
@@ -54,33 +52,35 @@ public static void main(String[] args) {
 
         //Car service
         Car fiatCar = new Car("500", fiatMaker);
-        Car opelCar = new Car("Blitz", fiatMaker);
-        Car tavriaCar = new Car("Slavuta", tavriaMaker);
-        Car ladaCar = new Car("Granta", ladaMaker);
-        final Manufacturer fordMaker = new Manufacturer("Ford", "USA");
+        Car opelCar = new Car("Blitz", opelMaker);
+        final Car tavriaCar = new Car("Slavuta", tavriaMaker);
+        final Car ladaCar = new Car("Granta", ladaMaker);
+        Manufacturer fordMaker = new Manufacturer("Ford", "USA");
+        manufacturerService.create(fordMaker);
         carService.create(fiatCar);
         carService.create(opelCar);
         carService.create(tavriaCar);
         carService.create(ladaCar);
         System.out.println(carService.get(1L).toString());
         System.out.println(carService.get(2L).toString());
         System.out.println(carService.get(3L).toString());
-        fiatCar.setManufacturer(fordMaker);
         carService.update(fiatCar);
         System.out.println(fiatMaker.toString());
         System.out.println(carService.delete(1L));
-        System.out.println(carService.delete(1L));
         System.out.println(carService.getAll());
 
         //Car service addDriverToCar
-        carService.addDriverToCar(opelDriver, opelCar);
         System.out.println("Adding driver to Opel");
         System.out.println(opelCar.toString());
+        System.out.println(opelDriver.toString());
+        carService.addDriverToCar(opelDriver, opelCar);
+        System.out.println(opelCar.toString());
```

Comment:
if you really want to track updates, you should better get the car from the db

Reply:
changed to get by id through a service